pkg/serializer/internal/metrics/service_checks_test.go

```diff
@@ -151,6 +154,20 @@ func createServiceChecks(numberOfItem int) ServiceChecks {
 	return ServiceChecks(serviceCheckCollections)
 }
 
+func decompressPayload(payload []byte) ([]byte, error) {
```

Comment:
This appears to be unused?

Reply:
This function is used for example at https://github.com/DataDog/datadog-agent/blob/7d83ee116fdc2c94e42b6fb8739eaa585deed1eb/pkg/serializer/internal/metrics/series_test.go#L313.